docs/Systems/Tailwind/UtilityClassReferences/Borders/border-spacing.stories.tsx

```diff
@@ -16,41 +15,42 @@ const classEntries: Array<{ utilityClassName: string; cssProperty: string }> =
   )
 
 export default {
-  title: `${CATEGORIES.tailwind}/Utility Class References/Borders/Border Spacing`,
-  component: <div>Hello</div>,
+  title: "Systems/Tailwind/Utility Class References/Borders/Border Spacing",
   parameters: {
     docs: {
       description: {
         component: utilityDescription(prefix, classEntries[0].utilityClassName),
       },
     },
   },
-}
+} as Meta
 
-const StickerSheetTemplate: Story<{ isReversed: boolean }> = ({
+const StickerSheetTemplate: StoryFn<{ isReversed: boolean }> = ({
   isReversed,
 }) => (
   <UtilityClassTemplate
     compiledCssPropertyName="border-spacing"
     classKeyValues={classEntries}
     renderExampleComponent={(utilityClass): React.ReactElement => (
       <table className={utilityClass}>
-        <tr>
-          <td className={classnames("outline-[black] outline-1 outline")}>
-            Tutant
-          </td>
-          <td className={classnames("outline-[black] outline-1 outline")}>
-            Meenage
-          </td>
-        </tr>
-        <tr>
-          <td className={classnames("outline-[black] outline-1 outline")}>
-            Neetle
-          </td>
-          <td className={classnames("outline-[black] outline-1 outline")}>
-            Teetles
-          </td>
-        </tr>
+        <tbody>
+          <tr>
+            <td className={classnames("outline-[black] outline-1 outline")}>
```

Comment:
Need to review all these usages of `classnames` with Tailwind classes, it shouldn't really be necessary.

Reply:
Oh very true. Definitely not needed unless it's something like this:
```
classnames(
	"outline-[black]", 
	"outline-1",
	"outline",
)
```